experimental/dds/tree2/src/feature-libraries/forestRepairDataStore.ts

```diff
@@ -62,6 +71,10 @@ export class ForestRepairDataStore implements RepairDataStore {
 			}
 		};
 
+		/**
+		 * Requires that `cursor` be positioned on a field.
```

Comment:
I think it would be better to say: 
```suggestion
		 * Visits the field `cursor` is positioned at.
```

since nothing currently says what field this visits.